### PR TITLE
FFT: Spc 414 reconnect when stream suddenly stops

### PR DESCRIPTION
**Pull request by BrianVanB**
It works, but there are a lot of but's.

If the forwarder comes back online and the processor reconnects and starts sending boxes again before the interface reloads, the sync _should_ be fine

If the forwarder comes back and the interface reloads before the processor starts sending boxes again, sync seems inconsistent. Sometimes it's fine, other times there is a small desync. Desync can be fixed fairly reliably by manually pausing the stream for a few seconds.
I think this could be fixed with the 'hack' that makes the video jump a little bit after loading. But this was removed earlier because the jump was considered annoying and not a good fix.

If the forwarder is not back yet when the interface reloads, there is a chance of ending up with a videojs error and will require a full page reload to fix.

TL:DR: As long as the processor is sending boxes before the interface reloads sync _should_ remain acceptable

IMO it's at least better than nothing.

### CODE REVIEW

**Comment by GerardvSchie** [comment]
Is there any failurerate, how many times can we expect the full refresh is needed

**Review by RienkF — APPROVED** [review]

**Comment by BrianVanB** [comment]
> Is there any failurerate, how many times can we expect the full refresh is needed

Unfortunately if the forwarder is not back up yet when the player is reloaded then it's pretty much guaranteed to need a full refresh once the forwarder is back up. If the forwarder is running then it should never need a full refresh.

**Review by GerardvSchie — APPROVED** [review]
